crates/turborepo-lib/src/commands/daemon.rs

```diff
@@ -193,6 +196,26 @@ async fn clean(
     }
 }
 
+async fn clean_logs(log_folder: &AbsoluteSystemPath) -> Result<(), DaemonError> {
```

Comment:
I don't think this needs to be async
```suggestion
fn clean_logs(log_folder: &AbsoluteSystemPath) -> Result<(), DaemonError> {
```